hg-fast-export.py

```diff
@@ -330,7 +330,7 @@ def verify_heads(ui,repo,cache,force):
   # get list of hg's branches to verify, don't take all git has
   for _,_,b in l:
     b=get_branch(b)
-    sha1=get_git_sha1(b)
+    sha1=get_git_sha1(sanitize_name(b))
```

Comment:
Good catch, you are credited in b7cc6ab3bfecaea47105c23739b289461c2bf347 which is basically your change, but following some refactoring.